gensim/models/wrappers/fasttext.py

```diff
@@ -270,26 +278,34 @@ def load_model_params(self, file_handle):
         self.wv.max_n = maxn
         self.sample = t
 
-    def load_dict(self, file_handle):
-        (vocab_size, nwords, _) = self.struct_unpack(file_handle, '@3i')
+    def load_dict(self, file_handle, encoding='utf8'):
+        vocab_size, nwords, _ = self.struct_unpack(file_handle, '@3i')
```

Comment:
I'd prefer keeping the changes related to the issue with the french wiki in a separate PR. We don't want those changes to block this PR from being merged.
In general, I think it's good practice to keep fixes for different issues in different branches/PRs.